### ISSUE TITLE
Model can not converge

### ISSUE THREAD

**Issue by VJJJJJJ1** [jishengpeng/WavTokenizer#52]
hi, I try to train wavtokenizer on ljspeech and libritts respectively, but the loss seems can not converge. I did'n change any code, just change the following configuration:
```
data:
  class_path: decoder.dataset.VocosDataModule
  init_args:
    train_params:
      filelist_path: ./WavTokenizer-main/data/LJSPEECH/train.txt
      sampling_rate: 24000
      num_samples: 24000
      batch_size: 16
      num_workers: 8

    val_params:
      filelist_path: ./WavTokenizer-main/data/LJSPEECH/val.txt
      sampling_rate: 24000
      num_samples: 24000
      batch_size: 5 
      num_workers: 8
```
and can see loss as the pic, Is my configuration incorrect? Or still need to continue to wait? thanks for reply!
![image](https://github.com/user-attachments/assets/97fb2d3e-9a31-4e6f-ae9c-d1dd252e9cf6)

**Comment by jishengpeng** [comment]
> hi, I try to train wavtokenizer on ljspeech and libritts respectively, but the loss seems can not converge. I did'n change any code, just change the following configuration:
> 
> ```
> data:
>   class_path: decoder.dataset.VocosDataModule
>   init_args:
>     train_params:
>       filelist_path: ./WavTokenizer-main/data/LJSPEECH/train.txt
>       sampling_rate: 24000
>       num_samples: 24000
>       batch_size: 16
>       num_workers: 8
> 
>     val_params:
>       filelist_path: ./WavTokenizer-main/data/LJSPEECH/val.txt
>       sampling_rate: 24000
>       num_samples: 24000
>       batch_size: 5 
>       num_workers: 8
> ```
> 
> and can see loss as the pic, Is my configuration incorrect? Or still need to continue to wait? thanks for reply! ![image](https://private-user-images.githubusercontent.com/77922137/385534657-97fb2d3e-9a31-4e6f-ae9c-d1dd252e9cf6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0ODAzMzUsIm5iZiI6MTczMTQ4MDAzNSwicGF0aCI6Ii83NzkyMjEzNy8zODU1MzQ2NTctOTdmYjJkM2UtOWEzMS00ZTZmLWFlOWMtZDFkZDI1MmU5Y2Y2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEzVDA2NDAzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxN2JlZmZhY2U0MTExMTNmYzE2NTBiOTVjYmY1ZTVlZGVmNzZlY2UzNWJjOGY0MGRmNWMwYzBhZmZmZDUyNGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HuTy5iP9dwddRaiNVP6PIExqrnxCavm_nhf07fEwR6g)

The TensorBoard visualization suggests that the training has stalled. I’ve encountered similar issues before, which are often due to problems with the model configuration or architecture. Based on the data configuration you provided, there doesn't appear to be any obvious issues. It might be helpful to start by training on the LibriTTS dataset and progressively debug from there. Typically, WavTokenizer converges quickly, and within a few hundred steps, the loss should already be in the double digits.

**Comment by VJJJJJJ1** [comment]
> The TensorBoard visualization suggests that the training has stalled. I’ve encountered similar issues before, which are often due to problems with the model configuration or architecture. Based on the data configuration you provided, there doesn't appear to be any obvious issues. It might be helpful to start by training on the LibriTTS dataset and progressively debug from there. Typically, WavTokenizer converges quickly, and within a few hundred steps, the loss should already be in the double digits.

I reduce half of the learning rate, and the model seems to start converging at the 900 steps. Thanks for reply!
